dashboard/app/controllers/pd/workshop_daily_survey_controller.rb

```diff
@@ -64,16 +64,14 @@ def new_pre_foorm
     end
 
     # General post-workshop survey using foorm system.
-    # GET '/pd/workshop_post_survey?enrollmentCode=code'
```

Comment:
Why remove this?

Reply:
I removed this to try and simplify the logic a bit, plus this method was labeled with `foorm` which would be misleading for the new non-foorm survey. There's the method `new_post` which has a super similar controller action of `GET /pd/workshop_survey/post/:enrollment_code` and just returns `new_post_foorm` anyways so I just made `new_post` the 'get survey by enrollment code' action in the controller. This way, whenever a survey is obtained by the enrollment code it always uses the same method (and within that, it determines whether to use foorm or not).